src/test/suite/fixtures/recorded/languages/python/chuckValue.yml

```diff
@@ -0,0 +1,23 @@
+spokenForm: chuck value
+languageId: python
+command:
+  actionName: remove
+  partialTargets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: value, includeSiblings: false}
+  extraArgs: []
+marks: {}
+initialState:
+  documentContents: "foo: str = \"hello\""
+  selections:
+    - anchor: {line: 0, character: 2}
+      active: {line: 0, character: 2}
+finalState:
+  documentContents: "foo: str "
```

Comment:
We should prob clean up this trailing string too, but prob best to leave that one for object-oriented targets refactor